src/Relay/Connection/ConnectionBuilder.php

```diff
@@ -245,7 +253,7 @@ public function getOffsetWithDefault(?string $cursor, int $defaultOffset): int
      */
     public function offsetToCursor($offset): string
     {
-        return \base64_encode(static::PREFIX.$offset);
+        return $this->cursorEncoder->encode(self::PREFIX.$offset);
```

Comment:
maybe we should give the prefix as an argument so the encoder could choose to use it or not?

Reply:
The prefix in this case is an implementation detail of the builder, so moving its handling outside the class imho is wrong

Reply:
ok nice :+1: